### DISCUSSION TITLE
Loading bars in bottom right corner missing

### DISCUSSION THREAD

**Discussion by Kerwood** [neoclide/coc.nvim#4318] [Q&A] [answered]
Hi. 
When installing coc.nvim using the `v0.0.82` tag, I get such nice loading bars at the bottom of the window when opening a Rust project. See screenshot below.
If I install coc from the release branch, the loading bars are gone. Are there a setting where I can enable these bars again?

![image](https://user-images.githubusercontent.com/4029521/197344427-01399942-603c-4f6d-b178-2ac4844f342a.png)

**Comment by fannheyward** [comment, marked as answer]
The progress notifications has change to display on status line by default. You can `"notification.statusLineProgress": false` to display them in floating windows.

**Reply by Kerwood** [reply]
Perfect, thanks :+1:
